### PR TITLE
Simplify

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: server/controllers/stream.js [hunk 1]
```diff
@@ -149,10 +108,7 @@ module.exports = function(req, res, next) {
 				res.set('Content-Type', 'text/xml');
 				res.send(streamToFeed(title, 'http://next.ft.com', streamPath, stream.items));
 			} else if (accepts === 'html') {
-
-				stream.currentNavItem = res.locals.nav.find(function(item) {
-					return item.id === req.params.thingId;
-				});
+				stream.currentNavItem = res.locals.nav.find(function(item) { return item.id === req.params.thingId; });
```
Comment (i-like-robots) [inline server/controllers/stream.js]: Maybe assign the `children` property to `currentNavItem` here to simplify the template.

**Comment by i-like-robots** [comment]
I definitely prefer the article transformation living in its own module.

**Comment by matthew-andrews** [comment]
own module or own repo?

**Comment by i-like-robots** [comment]
I was complimenting your choice of code reorganisation! I think the lack of emoji made the comment ambiguous.

This was doesn't: :+1:

**Comment by matthew-andrews** [comment]
emoji is important!  ok thanks i've implemented your suggestion, i think

**Comment by i-like-robots** [comment]
Exactly that, it's .1% nicer now =)

**Pull request merged**